### DISCUSSION TITLE
Can't update apktool in kali... Help needed!!

### DISCUSSION THREAD

**Discussion by aloo-bhujia** [iBotPeaches/Apktool#3521] [Q&A]
Having deprecated version of apktool (2.6.0) which is default on my kali machine, as whenever i try to  update it the newer versions is installed but when I use 'apktool --version' it shows 2.6.0 to which i uninstalled apktool and then i ran it again hoping to find a output such as "apktool is not installed" but instead it shows as apktool is still there, to which i went on the official website to install it  manually but still the version is the same. I have tried to fix it but no luck. Github is my last hope.

**Comment by iBotPeaches** [comment]
Kali tracks its own version - so its not exactly easy to update it. Instead I would recommend just downloading the jar and executing it manually.

Thus you could do like `java -jar apktool.jar` and execute it manually.

In terms of getting the Apktool version up to date in Kali - I was unaware my conversion to Kotlin would have messed it up so much. So that is still in discussion on best path forward.

**Comment by morox64** [comment]
If you install the FatRat, go to the following path ../TheFatRat/tools/apktool/ and delete the apktool and apktool.jar scripts and reinstall apktool normally from this website https://apktool.org/docs/install/#linux and it will work successfully

**Reply by hwac121** [reply]
Thank you morox64, damn if that didn't work! LOL
I was beating my head on the desk, trying everything.
Thank you thank you!
